pyproject.toml

```diff
@@ -12,7 +12,7 @@ torch = { version = ">=1.13.1", optional = true }
 transformers = { version = ">=4.26.1", optional = true }
 sentence-transformers = { version = ">=2.0.0", optional = true }
 wget = "^3.2"
-mmh3 = "^3.1.0"
+mmh3 = "^3.1.0 || ^4.0.0"
```

Comment:
You van simply do `>3.1.0`, and then it's anything bigger than 3.1, including 4.0